ui.frontend/src/view/FormFileInputWidgetTwo.js

```diff
@@ -0,0 +1,207 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import FormFileInputWidget from "./FormFileInputWidget";
+
+/**
+ * This class is responsible for interacting with the file input widget. It implements the file preview,
+ * file list, handling invalid file size, file name, file mime type functionality
+ */
+
+ class FormFileInputWidgetTwo extends FormFileInputWidget {
```

Comment:
Can we rename the classes. Something like FileInputWidget extends FileInputWidgetBase

Reply:
Renamed!